### DISCUSSION TITLE
"Optional" Dependencies for CLI

### DISCUSSION THREAD

**Discussion by CaptainInler** [kraina-ai/quackosm#130] [Q&A] [answered]
The [Documentation lists some Dependencies](https://kraina-ai.github.io/quackosm/0.8.1/#dependencies) as optional.
As far as I can see, these optional dependencies are all required for the CLI. Some are even marked as required for the CLI, others, like python-geohash, are not obviously required.
Is it intended that all dependencies are required for the CLI? If so, I will try to update the documentation accordingly.

**Comment by RaczeQ** [comment, marked as answer]
Hi, it is true that all optional dependencies are only required for CLI currently.
I forgot to copy this information for all listed packages in readme (missing in s2 and geohash).
